### ISSUE TITLE
Update page opens every time browser is launched

### ISSUE THREAD

**Issue by Bjornskjald** [sindresorhus/refined-github#1158]
Even when there's no update, I'm still seeing the update page

**Comment by paulirish** [comment]
dupe of #1147 i believe.

**Comment by Bjornskjald** [comment]
@paulirish That's correct, but that issue is closed and the page is still opening.

**Comment by paulirish** [comment]
Yes agree. I'm happy you filed it, just linking up the other so folks know it's fixed on master. :)

**Comment by bfred-it** [comment]
> Even when there's no update, I'm still seeing the update page

I just relaunched the browser and that does not seem to happen in Chrome nor in Firefox. It only happens when there's an update (almost daily), which is what likely what you got when you reopened the browser.
